docs/staking/intro.rst

```diff
@@ -0,0 +1,312 @@
+
+::
+
+    go get github.com/cosmos/gaia
+    cd $GOPATH/src/github.com/cosmos/gaia
+    make get_vendor_deps
+    make install
+
+It has three primary commands:
+
+::
+
+    Available Commands:
+      node        The Cosmos Network delegation-game blockchain test
+      rest-server REST client for gaia commands
+      client      Gaia light client
+                        
+      version     Show version info
+      help        Help about any command
+
+and a handful of flags that are highlighted only as necessary.
+
+The ``gaia node`` command is a proxt for running a tendermint node. You'll be using
+this command to either initialize a new node, or - using existing files - joining
+the testnet. 
+
+The ``gaia rest-server`` command is used by the `cosmos UI <https://github.com/cosmos/cosmos-ui>`__.
+
+Lastly, the ``gaia client`` command is the workhorse of the staking module. It allows
+for sending various transactions and other types of interaction with a running chain.
+The rest of this tutorial will cover several commands from ``gaia client`` and assume
+that you've setup or joined a testnet.
+
+Generating Keys
+---------------
+
+First, let's generate a key named ``alice``:
+
+::
+
+    gaia client keys new alice
+
+This will output:
+
+::
+
+    Enter a passphrase:
+    Repeat the passphrase:
+    alice	    5D93A6059B6592833CBC8FA3DA90EE0382198985
+    **Important** write this seed phrase in a safe place.
+    It is the only way to recover your account if you ever forget your password.
+
+    inject position weather divorce shine immense middle affair piece oval silver type until spike educate abandon
+
+which has your address and will be re-used throughout this tutorial.
+We recommend doing something like ``MYADDR=<your address>``. Writing 
+down the recovery phrase is crucial for production keys, however,
+for this tutorial you can skip this step.
+
+Because ``alice`` will be the initial validator, we need another key, ``bob`` who will first receives tokens from ``alice``, then declare candidacy as a validator. We also need an account for ``charlie`` who will bond and unbond to ``bob``.
+
+::
+
+    gaia client keys new bob
+    gaia client keys new charlie
+
+Now we can see the keys we've created:
+
+::
+
+    gaia client keys list
+
+which shows something like:
+
+::
+
+    All keys:
+    alice           5D93A6059B6592833CBC8FA3DA90EE0382198985
+    bob             5A35E4CC7B7DC0A5CB49CEA91763213A9AE92AD6
+    charlie         48F74F48281C89E5E4BE9092F735EA519768E8EF
+
+Try adding the ``--output json`` flag to the above command to get more information.
+The information for these keys is stored in ``~/.cosmos-gaia-cli``.
+We've got our keys made, so let's move on to the next step.
+
+Sending Tokens
+--------------
+
+We'll have ``alice`` who is currently quite rich, send some ``fermions`` to ``bob``:
+
+::
+
+    gaia client tx send --amount=1000fermion --sequence=1 --name=alice --to=5A35E4CC7B7DC0A5CB49CEA91763213A9AE92AD6
```

Comment:
Sorry just realized that if you get tokens from the faucet you only receive 10 https://www.cosmosvalidators.com/
As such we should really make the tutorial revolve around that - maybe scale everything down to 7 tokens (and give it a whirl to make sure then math adds up)